src/pages/authentication/signup/Signup.tsx

```diff
@@ -15,7 +15,6 @@ import { AppDispatch } from "../../../app/store";
 import { toast } from "react-toastify";
 import { unwrapResult } from "@reduxjs/toolkit";
 
-
 export const Signup = () => {
   const navigate = useNavigate();
   const dispatch = useDispatch<AppDispatch>();
```

Comment:
You can use the hook useAppDispatch which is defined in app/hooks file

Reply:
done